src/olcs/contrib/manager.js

```diff
@@ -104,11 +118,21 @@ olcs.contrib.Manager = class {
    * @return {Promise.<olcs.OLCesium>}
    */
   load() {
-    if (!this.promise_) {
+    if (!this.waitForLoad_) {
       const cesiumLazyLoader = new olcs.contrib.LazyLoader(this.cesiumUrl_);
-      this.promise_ = cesiumLazyLoader.load().then(() => this.onCesiumLoaded());
+      this.waitForLoad_ = true;
+      this.loadPromiseResolver(cesiumLazyLoader.load().then(
+          () => this.onCesiumLoaded())
+      );
     }
-    return this.promise_;
+    return this.loadPromise_;
+  }
+
+  /**
+   * @return {Promise.<olcs.OLCesium>}
+   */
+  getLoadPromise() {
```

Comment:
To keep the API smaller, I would rather add a parameter to the `load` method: `load(lazyLoad=true)`.
Then this new method would not be necessary.